source/org/zfin/db/postGmakePostloaddb/1169/ZFIN-9663.sql

```diff
@@ -0,0 +1,12 @@
+--liquibase formatted sql
+--changeset rtaylor:ZFIN-9663
+
+update updates set rec_id = 'ZDB-COMPANY-120808-1' where rec_id = 'ZDB-COMPANY-181130-1';
+update int_data_supplier set idsup_supplier_zdb_id = 'ZDB-COMPANY-120808-1' where idsup_supplier_zdb_id = 'ZDB-COMPANY-181130-1';
+
+delete from company where zdb_id = 'ZDB-COMPANY-181130-1';
```

Comment:
Is there no delete cascade for company on the zdb_active_data?

Reply:
Good point. I removed the unnecessary delete.